pkg/ccl/changefeedccl/kcjsonschema/kafka_connect_json_schema.go

```diff
@@ -225,3 +212,176 @@ func typeToSchema(typ *types.T) (Schema, error) {
 
 	}
 }
+
+// TestingMatchesJSON is a testing helper that asserts that the given data matches
+func (s Schema) TestingMatchesJSON(data any) error {
```

Comment:
Instead of making this a receiver, could we change it to a non-receiver function that accepts the struct as an argument? Feels a little weird to have a testing-only receiver on a production struct.

Reply:
I dont really agree that this is materially better but ok